build-system/babel-config/eslint-config.js

```diff
@@ -37,9 +37,14 @@ function getEslintConfig() {
   };
 }
 
+/**
+ * @return {{
+ *  manipulateOptions: {Function(_opts: *, parserOpts: *): void}
+ * }}
```

Comment:
TIL JSDoc types can be split across multiple lines like this. (In the past, I remember closure complaining about multiline types in runtime code. Probably not an issue when we start checking build system code with TS.)